napi/transform/CHANGELOG.md

```
@@ -4,6 +4,36 @@ All notable changes to this package will be documented in this file.
+### Bug Fixes
+
+- 4795059 transformer_plugins: Provide reference data when identifiers are replaced (#10620) (Boshen)
+
+### Performance
+
+- 88249f7 napi/transform: Do not create temp `String` (#10752) (overlookmotel)
+
+### Refactor
+
+
+## [0.68.0] - 2025-05-03
```

markdownlint-cli2 (0.17.2)

Multiple headings with the same content
null

(MD024, no-duplicate-heading)